docs/docs/overview/quickstart/get-started-with-nestjs.md

```diff
@@ -0,0 +1,286 @@
+---
+sidebar_position: 10
```

Comment:
I think this position will now have to be 11.

Reply:
updated to 12, because I see the other quickstart guide is currently 11